src/routes/(app)/news/SmallArticleCard.svelte

```diff
@@ -1,91 +1,62 @@
-            class="rounded-2xl"
-            src={article.imageUrl}
-            alt={article.header}
-          />
-        </figure>
-      {/if}
+<div class="flex flex-col overflow-hidden rounded-xl bg-base-200">
+  <div class="flex justify-between gap-4 bg-base-300 p-4">
+    <div class="flex flex-wrap gap-2">
+      {#each article.tags as tag}
+        <span class="badge text-xs text-neutral-400">{tag.name}</span>
+      {/each}
     </div>
-  {/if}
+  </div>
 
-  <div class="flex flex-row flex-wrap gap-2">
-    {#each tags as tag}
-      <a
-        href="?tags={encodeURIComponent(tag.name)}"
-        class="opacity-80 transition-opacity hover:opacity-100"
-        on:click={() => onTagClick?.(tag)}
-      >
-        <TagChip {tag} />
+  <div class="p-8">
+    <div class="flex-1">
+      <a href={"news/" + article.slug} class="group">
+        <h1 class="text-2xl font-bold group-hover:underline">
+          {article.header}
+        </h1>
+        <div class="prose mb-8 mt-2 line-clamp-3 prose-headings:text-sm">
+          <!-- eslint-disable-next-line svelte/no-at-html-tags -- Sanitized client-side -->
+          {@html marked(DOMPurify.sanitize(article.body))}
+        </div>
       </a>
-    {/each}
-  </div>
-  <div class="mt-4 flex flex-row flex-wrap items-start justify-between gap-2">
-    <div class="flex flex-col gap-2">
-      <LikersList {likers} />
-      <LikeButton {likers} {likeForm} articleId={article.id} />
     </div>
-    {#if commentCount > 0}
-      <a
-        href="/news/{article.slug}#comment-section"
-        class="link text-sm opacity-40 hover:opacity-60"
-      >
-        {commentCount} kommentarer
+
+    <div class="flex gap-4">
+      <a href={"members/" + article.author.member.studentId}>
+        <img
+          class="size-10 rounded-full object-cover"
+          src={article.author.member.picturePath ??
+            "https://gravatar.com/avatar?s=100&d=mp"}
+          alt={article.author.member.firstName}
+        />
       </a>
-    {/if}
-    <div class="m-2">
-      <AuthorSignature
-        member={author.member}
-        position={author.mandate?.position}
-        customAuthor={author.customAuthor ?? undefined}
-        type={author.type}
-      />
+      <div class="flex w-full items-center justify-between gap-4">
+        <div>
+          <a href={"members/" + article.author.member.studentId}>
+            <h2 class="text-sm font-semibold text-primary hover:underline">
+              {getFullName(article.author.member)}
+            </h2>
+          </a>
+          {#if article.author.mandate}
+            <a href={"positions/" + article.author.mandate.position.id}>
+              <p class="my-1 text-xs font-light hover:underline">
+                {article.author.mandate.position.name}
+              </p>
+            </a>
+          {/if}
+        </div>
+        <p class="my-1 self-end text-xs font-light text-neutral-600">
+          {article.createdAt.toLocaleDateString()}
+        </p>
```

Comment:
Would be nice if this could display "3 minutes ago", "2 hours ago", "1 day ago" and then the date, etc.

Reply:
Fixed